Source/Core/CesiumTerrainProvider.js

```diff
@@ -417,11 +425,19 @@ define([
 
         var skirtHeight = provider.getLevelMaximumGeometricError(level) * 5.0;
 
+        var rectangle = provider._tilingScheme.tileXYToRectangle(x, y, level);
+        var boundingOBB;
+        if (rectangle.width < CesiumMath.PI_OVER_TWO + 0.001) {
+            var tangentPlane = new EllipsoidTangentPlane(center, provider._tilingScheme.ellipsoid);
+            boundingOBB = OrientedBoundingBox.fromRectangleTangentPlane(rectangle, tangentPlane, minimumHeight, maximumHeight);
```

Comment:
This is awkward.  How is:
```javascript
boundingOBB = OrientedBoundingBox.fromRectangle(rectangle, minimumHeight, maximumHeight, provider._tilingScheme.ellipsoid);
```
Only `rectangle` would be required.  `EllipsoidTangentPlane` would be used in the OBB.  That dependency should be fine.